know/rdf/parser/wikidata_formatter.py

```diff
@@ -55,3 +55,45 @@ def format_date(val: str, key: str) -> str:
         return val
 
     return format_query(response, entries_translations, entries_links, format_date)
+
+def format_book(response: dict) -> dict:
+    """ Formats a book into the expected output format """
+    entries_translations = {
+        "authorLabel": "Author",
+        "name": "Title",
+        "genreLabel": "Genre",
+        "published": "Published"
+    }
+    entries_links = {
+        "authorLabel": "author"
+    }
+    def format_date(val: str, key: str) -> str:
+        # If the key is the publication date, format it into a date string
+        if val and key == "published":
+            return format_date_string(val)
+
+        # For all other keys, leave them alone
+        return val
+    return format_query(response, entries_translations, entries_links, format_date)
+def format_person(response: dict) -> dict:
+    """Format a person into the expected output format """

```

Comment:
```suggestion
    """ Format a person into the expected output format
          - William Blake: https://www.wikidata.org/wiki/Q41513
     """
```
add like ~3 examples for these for both book and person please! I'm going to do the same for landmark and country as well